### PR TITLE
Fix button appearance

### PR DESCRIPTION
**Pull request by neoOpus**
Now button appears as the default GH

### CODE REVIEW

File: github-rtl-comments.user.js [hunk 1]
```diff
@@ -55,9 +54,9 @@
 			toolbars = $$(".toolbar-commenting"),
 			indx = toolbars.length;
 		if (indx) {
-			button = document.createElement("button");
-			button.type = "button";
-			button.className = "ghu-rtl toolbar-item tooltipped tooltipped-n";
+			button = document.createElement("md-code");
+			button.type = "md-code";
```
Comment (Mottie) [inline github-rtl-comments.user.js:58]: Thanks for your work, but `md-code` is specific to the code button. Please change these two lines back to `button`.

**Review by Mottie — CHANGES_REQUESTED** [review]

**Comment by Mottie** [comment]
Thanks!

**Comment by neoOpus** [comment]
> Thanks!

Thank you for the merge and for letting me know about the md-code... right now I am trying to make this work with this extension [GitHub Writer - Chrome Web Store (google.com)](https://chrome.google.com/webstore/detail/github-writer/diilnnhpcdjhhkjcbdljaonhmhapadap) as it add some nice features too but breaks some userscripts including this one...

**Comment by Mottie** [comment]
Nice extension! I'll take a look when I have some free time (which is rare 😿)

**Pull request merged**